src/Tests/Backend/Backend.test.ts

```diff
@@ -48,44 +48,51 @@ class BackendMockup implements Backend {
 }
 
 suite("Backend", function () {
+  setup(function () {
+    Object.keys(globalBackendMap).forEach(
+      (key) => delete globalBackendMap[key]
+    );
+    Object.keys(gToolchainEnvMap).forEach(
+      (key) => delete gToolchainEnvMap[key]
+    );
+  });
+
+  teardown(function () {
```

Comment:
Could you move this teardown function to the bottom of this "Backend" test suite? There is absolutely no problem in your code but our company static analysis tool will spot it as a defect. (related to https://github.com/Samsung/ONE-vscode/issues/1678)

Reply:
Thank you for noticing me!! I updated Toolchain test code to move teardown downwards in https://github.com/Samsung/ONE-vscode/pull/1686/commits/153a61833e95f1ee7e6ff2fb97400f6fdadfa5ca 😄